packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md

```diff
@@ -117,9 +117,18 @@ You can also reference these org-level attributes, such as:
 
 All conditional logic that is supported by the Velocity Templating Engine, such as `if`, `elseif`, or `else` constructs and `foreach` loops, is available for you to use in your templates. See the [Velocity documentation](http://velocity.apache.org/engine/1.7/user-guide.html).
 
-#### Email templates for customization
+<ApiLifecycle access="ie" /><br>
+
+#### Access to app context
+
+Okta Identity Engine orgs have access to app context within emails using the Velocity Templating Language. When an end user enters an authentication flow, OIE stores the app context in the state token. The following properties are available on the app context:
+
+* `$app.id`
+* `$app.name`
+* `$app.label`
+
+When used with conditional logic, you can trigger per app branding and define strings to uniquely customize an email template based on the app your end user was using when the email was triggered. App context is not available on Classic Engine since the state token does not exist there.
```

Comment:
```suggestion
When these properties are used with conditional logic, you can trigger branding for the specified app and define strings to uniquely customize an email template based on the app from where the email was triggered. App context is not available on Classic Engine since the state token does not exist there.
```